api/services/ocean/spark/schemas/internalId.yaml

```diff
@@ -0,0 +1,4 @@
+type: string
+example: "fa61bb92-4bb7-49aa-87d3-7823bd263d1e"
+description: >
+  An internal ID describing the entity. As a user of Ocean Spark, you do not need this ID.
```

Comment:
Can we omit the internalId from the API response when calls are made by end users? 
So not to confuse them. This is something we could do later, not necessarily needed for right now.

>"internalId": "fa61bb92-4bb7-49aa-87d3-7823bd263d1e",
"id": "daily-reporting-2021-08-18",
"displayName": "Daily Reporting 2021-08-18",
"accountId": "act-123456789",
"organizationId": 606012341234,
"userId": 59438,
"clusterId": "osc-20fac3f1",
"controllerClusterId": "my-ocean-cluster",

Note: In a previous work experience we had certain API fields that had to be provided to the webapp, but which were for internal use only, and so we didn't want to show them to customers using the API. Our internal services would add a special auth token in their API requests. Unless this special token was there, the fields marked as internal were automatically stripped from the API.

Reply:
@crezvoy do you know if there is an internal service that uses the `internalId`?
Feel free to defer the question to someone else if you don't know the answer :)

Reply:
@crezvoy do you know if there is an internal service that uses the `internalId`?
Feel free to defer the question to someone else if you don't know the answer :)

Reply:
@geoffroynicolas maybe?

Reply:
Sorry for the double message. There's actually a thread about that here: https://spotinst.slack.com/archives/C020SQXLLKZ/p1637586169487500

Reply:
It seems we'll remove this field from the API. Let's keep it until it is actually removed though: we should explain what this is until it disappears.